Google.Api.Generator/Generation/ResourceNamesGenerator.cs

```diff
@@ -225,7 +251,7 @@ private IEnumerable<MethodDeclarationSyntax> FormatMethods()
                 }
             }
 
-            private IEnumerable<MethodDeclarationSyntax> ParseMethods()
+            private IEnumerable<MemberDeclarationSyntax> ParseMethods()
```

Comment:
I don't understand why this declaration needs to change - you're not yielding any non-methods, are you?

Reply:
Oh yes, you're right.
For a while it was going to emit an extra field, but I ended up not doing that.
I'll revert this change.